micrometer-jakarta/build.gradle

```diff
@@ -0,0 +1,116 @@
+plugins {
+    id 'me.champeau.mrjar' version "0.1.1"
+}
+
+description 'Module for Jakarta based instrumentations'
+
+jar {
+    bundle {
+
+        bnd '''\
+        Import-Package: \
+            org.eclipse.jetty.*;resolution:=dynamic;version="${@}",\
+            org.glassfish.jersey.*;resolution:=dynamic;version="${@}",\
+            ch.qos.logback.*;resolution:=dynamic;version="${@}",\
+            org.apache.logging.log4j.*;resolution:=dynamic;version="${@}",\
+            jakarta.jms.*;resolution:=dynamic;version="${@}",\
+            jakarta.servlet.*;resolution:=dynamic;version="${@}",\
+            io.micrometer.context.*;resolution:=dynamic,\
+            io.micrometer.observation.*;resolution:=dynamic;version="${@}",\
+            *
+        '''.stripIndent()
+    }
+}
+
+dependencies {
+    api project(":micrometer-core")
+    api project(":micrometer-commons")
+    api project(":micrometer-observation")
+    optionalApi 'jakarta.ws.rs:jakarta.ws.rs-api:3.+' // jakarta
+
+    // Aspects
+    optionalApi 'org.aspectj:aspectjweaver'
+
+    optionalApi 'org.eclipse.jetty:jetty-server'
+    // jakarta servlet
+    optionalApi 'jakarta.servlet:jakarta.servlet-api:5.+'
+    optionalApi 'org.eclipse.jetty:jetty-client:12.0.+'
+    optionalApi 'org.apache.tomcat.embed:tomcat-embed-core:10.1.+'
+    optionalApi 'org.glassfish.jersey.core:jersey-server:3.1.+'
```

Comment:
Shouldn't we add everything that has a version here to `dependencies.gradle`?

Reply:
No because we have tomcat in version 8 for micrometer-core and 10 for jakarta. I will double check if some of those can't be moved